tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentNegativeTest.java

```diff
@@ -37,7 +37,7 @@
     @BeforeClass
     public void setup() {
         compileResult = BCompileUtil.compile("test-src/annotations/annot_attachments_negative.bal");
-        Assert.assertEquals(compileResult.getErrorCount(), 253);
+        Assert.assertEquals(compileResult.getErrorCount(), 266);
```

Comment:
Do we already have positive tests for this?

Reply:
Added